.dockerignore

```diff
@@ -3,3 +3,6 @@
 
 .*
 *.egg*
+**/__pycache__/
+**.pyc
+**.pyo
```

Comment:
I was able to reproduce @acmiyaguchi's issue on my machine and then solve the problem by using `**` to ignore these caches in arbitrary subdirectories.